Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -3363,16 +3361,20 @@ CharacterOffset AXObjectCache::characterOffsetFromVisiblePosition(const VisibleP
     return result;
 }
 
-AccessibilityObject* AXObjectCache::accessibilityObjectForTextMarkerData(const TextMarkerData& textMarkerData)
+AccessibilityObject* AXObjectCache::objectForTextMarkerData(const TextMarkerData& textMarkerData)
 {
     if (textMarkerData.ignored)
         return nullptr;
 
-    RefPtr domNode = textMarkerData.node.get();
-    if (!domNode || !isNodeInUse(*domNode))
+    RefPtr object = m_objects.get(textMarkerData.axObjectID());
+    if (!object)
         return nullptr;
 
-    return getOrCreate(*domNode);
+    Node* node = object->node();
```

Comment:
Do we want to hold a strong reference to this node via RefPtr to make sure it stays valid for `getOrCreate`?

Reply:
In getOrCreate we are already doing:
Ref protectedNode { node };
So it would be refcount churn. The question moving forward is, do we always want to guaranty the survival of the parameter in the caller or the callee.